### DISCUSSION TITLE
NSPanel Pro 120 - installing latest Webview

### DISCUSSION THREAD

**Discussion by mcceeq** [seaky/nspanel_pro_tools_apk#151] [Q&A] [answered]
Hi all,

I'm trying to install a newish webview. Reason is, it seems some card_mod styles and HA sections don't work on older webviews. I have a 86 with xposed and webview 128 which works well, but when I install xposed on the 120, it fails to boot. I have installed 108 from this readme, but it's not new enough. When I use the same 128 as on the 86 it installs but doesn't actually show up. The same with the very latest 130. I'm using the arm64v8, armeabi-v7a apks. Any advice?

**Comment by seaky** [comment, marked as answer]
try this I uploaded some other versions 
https://community.home-assistant.io/t/nspanel-pro-custom-android-apps-working/459563/267?u=seaky

**Reply by mcceeq** [reply]
Thanks sneaky, I used 128 and works perfectly. Just for my info, what is different in your webview than the standard webview that it's seen by the os?

**Comment by maximus3331** [comment]
I have nspannel pro 120 vith 3.7.1 firmware. this firmware has 129 webview and HA Companion tems looks not correctly.
128 webview not installed 
![image](https://github.com/user-attachments/assets/362bb961-70ef-49fc-9323-baedfd2eddd4)
132 webview not installed directly "adb install"

**Comment by seaky** [comment]
try this one 
`adb install -r -d` 

or the version 131
https://www.apkmirror.com/apk/lineageos/android-system-webview-2/android-system-webview-2-131-0-6778-200-release/android-system-webview-131-0-6778-200-9-android-apk-download/

**Reply by Niek** [reply]
This one worked very well for me, thanks.

**Reply by Niek** [reply]
Newer version:
https://www.apkmirror.com/apk/lineageos/android-system-webview-2/android-system-webview-138-0-7204-63-2-release/android-system-webview-138-0-7204-63-8-android-apk-download/download/?key=279103ea1e4a8c60bda489bbab0f738c6af71196

Here is the search URL showing the latest version (unfortunately 138 seems the latest release for Android 8):
https://www.apkmirror.com/?post_type=app_release&searchtype=apk&s=com.android.webview&minapi-max=26&arch%5B%5D=arm64-v8a&bundles%5B%5D=apk_files
